plugins/inputs/slab/slab.go

```diff
@@ -0,0 +1,32 @@
+package slab
+
+import (
+	"os"
+	"path"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type SlabStats struct {
+	Log telegraf.Logger `toml:"-"`
+
+	statFile string
+	UseSudo  bool `toml:"use_sudo"`
+}
+
+func getHostProc() string {
+	procPath := "/proc"
+	if os.Getenv("HOST_PROC") != "" {
+		procPath = os.Getenv("HOST_PROC")
+	}
+	return procPath
+}
+
+func init() {
+	inputs.Add("slab", func() telegraf.Input {
+		return &SlabStats{
+			statFile: path.Join(getHostProc(), "/slabinfo"),
+		}
+	})
+}
```

Comment:
Since this plugin is only valid on linux, we should probably add linux-only build tags for this file too so the plugin isn't registered on other platforms. That way it won't show up in sample config files build for mac, windows, freebsd, etc.

Reply:
I have adapted build tags with reference to [conntrack input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/conntrack).